src/sql/workbench/services/connection/common/connectionContextKey.ts

```diff
@@ -14,12 +14,15 @@ export class ConnectionContextKey implements IContextKey<IConnectionProfile> {
 	static Database = new RawContextKey<string>('databaseName', undefined);
 	static Connection = new RawContextKey<IConnectionProfile>('connection', undefined);
 	static IsQueryProvider = new RawContextKey<boolean>('isQueryProvider', false);
+	static CanOpenInAzurePortal = new RawContextKey<boolean>('canOpenInAzurePortal', false);
```

Comment:
I guess I'm not sure how useful it'd be but generally I think it's better to avoid having super specific keys like this and instead make separate ones for each property - or at the very least name it something more generic like 'HasAzurePortalProperties'

Reply:
Hmm Just so you know I did get this other PR in and that has the same key.

Would you recommend I switch that one as well?

https://github.com/microsoft/azuredatastudio/pull/10535

If so I would prefer to get this merged in and then make a follow up PR on that changing the keys for both of these areas.

Reply:
Since you've approved it I'm going to merge it in. Let me know if you want me to do that followup task!

Reply:
Yup, that's all good. I'll sign off if I don't really have a strong opinion on something - such as in this case - and so if you decide to leave it as is that's fine. If you agree with what I said go ahead and change it but I don't really have a problem with the current implementation so it's fine to keep as is unless we find a better reason to switch.

Reply:
I'll do some playing around with it and make a new PR if I find a cleaner way to do this!